tests/test_scripts/unit_tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+sudo R \
+  -e "getwd();" \
+  -e "devtools::install();" \
+  -e "devtools::build();" \
+  -e "devtools::load_all();" \
+  -e "res <- testthat::test_dir('../testthat/unit_tests', reporter='summary');" \
```

Comment:
Is it customary when using this testing framework to use 'testthat' as the test directory as opposed to something more generic like 'tests'?

Reply:
Just realized that this is already a subdir of tests. Any reason to not put to use tests/unittests ?

Reply:
testthat R package forces you to use testthat as the root directory